airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -71,6 +71,9 @@ def __init__(
         aws_conn_id: str = "aws_default",
         steps: list[dict] | str | None = None,
         wait_for_completion: bool = False,
+        cancel_existing_steps: bool = True,
+        steps_states: list[str],
```

Comment:
This parameter is mandatory right now, which also might broke someone pipeline

Reply:
Handled